bitnami/postgresql-ha/README.md

```diff
@@ -173,6 +173,8 @@ The following table lists the configurable parameters of the PostgreSQL HA chart
 | `metrics.serviceMonitor.namespace`             | Optional namespace which Prometheus is running in                                                                                                                    | `nil`                                                        |
 | `metrics.serviceMonitor.interval`              | How frequently to scrape metrics (use by default, falling back to Prometheus' default)                                                                               | `nil`                                                        |
 | `metrics.serviceMonitor.selector`              | Default to kube-prometheus install (CoreOS recommended), but should be set according to Prometheus install                                                           | `{prometheus: "kube-prometheus"}`                            |
+| `metrics.serviceMonitor.relabelings`           | ServiceMonitor relabelings                                                                                                                                           | `[]`                                                         |
+| `metrics.serviceMonitor.metricRelabelings`     | ServiceMonitor metricRelabelings                                                                                                                                     | `[]`                                                         |
```

Comment:
You're evaluating both parameters a template. Therefore, please reflect that:
```diff
- | `metrics.serviceMonitor.relabelings` | ServiceMonitor relabelings   | `[]`                                         |
+ | `metrics.serviceMonitor.relabelings` | ServiceMonitor relabelings   | `[]` (The value is evaluated as a template). |
```
Apply the same suggestion to both parameters